.github/scripts/check-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Checking if current tag matches the package version
+current_tag=$(echo $GITHUB_REF | tr -d 'refs/tags/v')
+file1='meilisearch-auth/Cargo.toml'
+file2='meilisearch-http/Cargo.toml'
+file3='meilisearch-lib/Cargo.toml'
+file4='meilisearch-types/Cargo.toml'
+file5='Cargo.lock'
```

Comment:
Actually, I was wondering, why are we bumping the version of meilisearch-[`auth`, `http`, `lib`, `types`] when the only interesting thing is the meilisearch-lib?

Maybe we should take a decision and decide if we;
1. Bump the version number of ALL the crate in the repo,  including `permissive-json-pointer`.
That would allow us to do a;
```suggestion
# Checking if current tag matches the package version
current_tag=$(echo $GITHUB_REF | tr -d 'refs/tags/v')
files='*/Cargo.toml'
lock_file='Cargo.lock'
```
2. Only bump `meilisearch-http` since it's the one that's going to set the version number of the binary.

---------

I don't know what's the best but here is a few examples of projects that do one or the other;
1. [helix-editor](https://github.com/helix-editor/helix/)
2. [ripgrep](https://github.com/BurntSushi/ripgrep/tree/master/crates), [tauri](https://github.com/tauri-apps/tauri/tree/dev/core), [serde](https://github.com/serde-rs/serde), [actix-web](https://github.com/actix/actix-web), [rust](https://github.com/rust-lang/rust/tree/master/library) (-> they make everything be in 0.0.0), [deno](https://github.com/denoland/deno)

Reply:
You're right, I forgot `permissive-json-pointer`: so far we are doing the `1` solution, so I need to add it.
I will update the code to do solution 1 first, and I will open the discussion with core-team!